test/DynamoCoreWpfTests/AnnotationViewTests.cs

```diff
@@ -69,34 +70,6 @@ public  void TestAnnotationMouseClickEvent()
 
             //The group and the node should be selected
             Assert.AreEqual(2, DynamoSelection.Instance.Selection.Count());
-
-            //Clear the selection
-            DynamoSelection.Instance.ClearSelection();
-           
-            //Change the Zoom
-            annotationView.ViewModel.WorkspaceViewModel.SetZoomCommand.Execute(0.4);
-
-            //Group textblock should be collapsed.
-            Assert.IsFalse(annotationView.GroupTextBlock.IsVisible);
-
-            //Change the Zoom
-            annotationView.ViewModel.WorkspaceViewModel.SetZoomCommand.Execute(0.5);
-
-            //Group textblock should be visible.
-            Assert.IsTrue(annotationView.GroupTextBlock.IsVisible);
-
-            //Change the Zoom
-            annotationView.ViewModel.WorkspaceViewModel.SetZoomCommand.Execute(0.4);
-
-            //Group textblock should be collapsed.
-            Assert.IsFalse(annotationView.GroupTextBlock.IsVisible);
-
-            //Now change the font size - note: group textblock is collapsed now
-            annotationView.ViewModel.FontSize = 48;
-
-            //Group textblock should be visible.
-            Assert.IsTrue(annotationView.GroupTextBlock.IsVisible);
```

Comment:
@QilongTang FYI modified this test as the visibility checks does not really apply anymore. Unless we want to turn the hide on specific zoom back

Reply:
Thanks I dont think that is required, @Amoursol @Jingyi-Wen can you confirm?

Reply:
Got confirmation with @Jingyi-Wen